mem_sql/pom.xml

```diff
@@ -259,12 +259,10 @@
                             <key>com.splicemachine.enableLegacyAsserts</key>
                             <value>true</value>
                         </systemProperty>
```

Comment:
This should only be uncommented while debugging in your local branch.  We shouldn't have this uncommented for everyone.  It will fill up the mem_sql folder with class files and depending on the queries run this could take a lot of space.